tests/test_regression.py

```diff
@@ -171,9 +171,12 @@ class Config:
 
         original = Config("test_dump")
 
-        with tempfile.NamedTemporaryFile() as f:
-            dataconf.dump(f.name, original, out="yaml")
-            validate = dataconf.file(f.name, Config)
+        temporary_file = tempfile.NamedTemporaryFile()
+        temporary_file_name = temporary_file.name
+        temporary_file.close()
```

Comment:
another windows specific behaviour, have a look [there](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) and see whether `delete=False` is enough? do not forget to clean the file

Reply:
delete=False solved it

Reply:
Can you add a manual remove after the test has run? like a try/finally

Reply:
I've added a fixture to provide temp file and a teardown which will close the file